lib/fortnox/api/repositories/base/savers.rb

```diff
@@ -9,19 +9,30 @@ def save( entity )
           hash = entity_to_hash( entity )
           json = hash.to_json
 
-          response = entity.new? ? save_new( json ) : update_existing( json )
+          entity.new? ? save_new( json ) : update_existing( hash )
```

Comment:
What about:
```
return save_new( json ) if entity.new?
update_existing( json )
```
You are converting to json even if you don't need it now and you are creating two sub methods with different levels of abstraction (one on the level of a hash and the other on the level of json).